Dockerfile

```diff
@@ -1,13 +1,12 @@
 FROM python:3.6.4-alpine3.7
 MAINTAINER tsu-shiuan@zappistore.com
 
-RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
-COPY requirements.txt /usr/src/app/
+COPY requirements.txt .
 RUN pip install --no-cache-dir -r requirements.txt
 
-ADD snapshooter.py /usr/src/app/snapshooter.py
-ADD helper_methods.py /usr/src/app/helper_methods.py
-ADD config.yml /usr/src/app/config.yml
+ADD snapshooter.py .
```

Comment:
It would be nice to use a `.dockerignore` as a whitelist and just add all the files to the image here. See [this](https://github.com/zacblazic/graffiti-monkey/blob/5b17510a57e5d506ac4163eaee6fc02b4f065271/.dockerignore) as an example.

Reply:
Addressed here: https://github.com/tsu-shiuan/snapshooter/pull/1/commits/32c779edd246dc97fe8d723cc3518f3907d078b2